krux_sqs/sqs.py

```diff
@@ -231,3 +231,36 @@ def send_messages(self, queue_name, messages):
                 q.send_messages(Entries=chunk)
         else:
             self._logger.debug('Message is empty. Not sending any messages.')
+
+    def send_messages_to_fifo(self, queue_name, messages, group_id):
```

Comment:
This is similar to the normal queue function except for the group_id part..

I think it would be okay to add an extra argument `group_id=None` to the previous function. and just handle the None case inside `entries.append` section

Reply:
I would do the same and requesting that change. @AmberZhang1230, if you want to, we can pair program this on tomorrow or Friday.